src/mmw/js/src/user/views.js

```diff
@@ -219,6 +239,14 @@ var LoginModalView = ModalBaseView.extend({
 
     // Login with ITSI
     itsiLogin: function() {
+        var selector = '.itsi-login',
+            model = this.model;
+
+        if (coreUtils.modalButtonDisabled(model, selector)) {
+            return;
+        }
+        coreUtils.modalButtonToggle(model, selector, false);
```

Comment:
No, it was just an oversight.  Since there is a return in the if-clause, there is no difference between putting this in an else and not.